editor/src/com/talosvfx/talos/editor/project2/apps/SceneEditorApp.java

```diff
@@ -86,5 +86,27 @@ public void onUpdate () {
 	public void GONameChangeCommand(GONameChangeCommand command) {
 		workspaceWidget.changeGOName(command.getGo(), command.getSuggestedName());
 	}
+
+	@Override
+	public void applyPreferences(Object appPreferences) {
+		super.applyPreferences(appPreferences);
+		SceneEditorAppPrefs prefs = (SceneEditorAppPrefs) appPreferences;
+		workspaceWidget.setCameraPos(prefs.cameraPos);
+		workspaceWidget.setCameraZoom(prefs.cameraZoom);
+	}
+
+	@Override
+	public Object getCurrentPreference() {
+		SceneEditorAppPrefs prefs = (SceneEditorAppPrefs) super.getCurrentPreference();
+		prefs.cameraPos = workspaceWidget.getCameraPos();
+		prefs.cameraZoom = workspaceWidget.getCameraZoom();
+		return prefs;
+	}
+
+	@AppPreference
+	private static class SceneEditorAppPrefs {
```

Comment:
This should extend some other class. ViewportViewSettings are in many apps, and it doesn't make sense to add this stuff just for SceneEditorApp and then copy paste to others

Reply:
Done